src/client/components/Sidebar/Topics.less

```diff
@@ -8,7 +9,7 @@
     font-weight: 500;
   }
 
-  & > h5 {
+  & > a {
     color: @text-color;
     margin: 12px 0;
```

Comment:
Ahrefs are inline elements and can't have margin. Because of that link changed its placement:
![image](https://user-images.githubusercontent.com/1968722/38824322-2a7f21ce-41a9-11e8-8ca8-3134c006b3ed.png)
vs
![image](https://user-images.githubusercontent.com/1968722/38824341-3708849e-41a9-11e8-87bd-df32c014d8ab.png)